lib/mayaUsd/ufe/UsdUIInfoHandler.cpp

```diff
@@ -68,6 +68,24 @@ void addMetadataCount(
         addMetadataStrings(refs.size(), tooltip, needComma, singular, plural);
     }
 }
+
+std::vector<double> getInvisibleColor()
+{
+    static std::vector<double> rgb;
+    static bool                initialized = false;
+    if (!initialized) {
```

Comment:
We cannot do this once because the outlinerInvisibleColor can be modified while Maya is running and we should respect that.

Reply:
Fixed it by registering a callback when the display colors change.